pkgs/development/python-modules/mongomock/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, pbr
+, sentinels
+, six
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mongomock";
+  version = "3.23.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1pdh4pj5n6dsaqy98q40wig5y6imfs1p043cgkaaw8f2hxy5x56r";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    pbr
```

Comment:
`pbr` is usually in `nativeBuildInputs` and not a run-time requirement. `setuptools` are also not mention in [`requirements.txt`](https://github.com/mongomock/mongomock/blob/3.23.0/requirements.txt]

Reply:
That depends on the usage. If pbr is used in the program itself which some openstack things do then it needs to be propagated.

Reply:
I haven't found any other reference to pbr except [here](https://github.com/mongomock/mongomock/blob/cabc43a6027c306302bc0457cbd6f6a179fe211c/setup.py#L4)
I'm not familiar with pbr, but looking at the [documentation](https://docs.openstack.org/pbr/latest/user/using.html) it seems, it is mainly (only?) used at build time.
I think its save to put it under `nativeBuildInputs`